runners/samza/src/test/resources/ExpectedDag.json

```diff
@@ -0,0 +1,97 @@
+              "shortName":"Window.Assign",
+              "enclosingNode":"Window.Into()",
+              "id":"Window.Into()/Window.Assign"}
+          ]},
+        { "fullName":"Combine.perKey(SumInteger)",
+          "shortName":"Combine.perKey(SumInteger)",
+          "id":"Combine.perKey(SumInteger)",
+          "enclosingNode":"OuterMostNode",
+          "ChildNode":[
+            { "fullName":"Combine.perKey(SumInteger)/GroupByKey",
+              "shortName":"GroupByKey",
+              "enclosingNode":"Combine.perKey(SumInteger)",
+              "id":"Combine.perKey(SumInteger)/GroupByKey"},
+            { "fullName":"Combine.perKey(SumInteger)/Combine.GroupedValues",
+              "shortName":"Combine.GroupedValues",
+              "id":"Combine.perKey(SumInteger)/Combine.GroupedValues",
+              "enclosingNode":"Combine.perKey(SumInteger)",
+              "ChildNode":[
+                { "fullName":"Combine.perKey(SumInteger)/Combine.GroupedValues/ParDo(Anonymous)",
+                  "shortName":"ParDo(Anonymous)",
+                  "id":"Combine.perKey(SumInteger)/Combine.GroupedValues/ParDo(Anonymous)",
+                  "enclosingNode":"Combine.perKey(SumInteger)/Combine.GroupedValues",
+                  "ChildNode":[
+                    { "fullName":"Combine.perKey(SumInteger)/Combine.GroupedValues/ParDo(Anonymous)/ParMultiDo(Anonymous)",
+                      "shortName":"ParMultiDo(Anonymous)",
+                      "enclosingNode":"Combine.perKey(SumInteger)/Combine.GroupedValues/ParDo(Anonymous)",
+                      "id":"Combine.perKey(SumInteger)/Combine.GroupedValues/ParDo(Anonymous)/ParMultiDo(Anonymous)"}
+                  ]}
+              ]}
+          ]}
+      ]}
+  ]
+,"graphLinks": [{"from":"Create.TimestampedValues/Create.Values/Read(CreateSource)/Impulse","to":"Create.TimestampedValues/Create.Values/Read(CreateSource)/ParDo(OutputSingleSource)/ParMultiDo(OutputSingleSource)","hashId":"402#bb20b45fd4d95138"},{"from":"Create.TimestampedValues/Create.Values/Read(CreateSource)/ParDo(OutputSingleSource)/ParMultiDo(OutputSingleSource)","to":"Create.TimestampedValues/Create.Values/Read(CreateSource)/ParDo(BoundedSourceAsSDFWrapper)/ParMultiDo(BoundedSourceAsSDFWrapper)","hashId":"402#3d93cb799b3970be"},{"from":"Create.TimestampedValues/Create.Values/Read(CreateSource)/ParDo(BoundedSourceAsSDFWrapper)/ParMultiDo(BoundedSourceAsSDFWrapper)","to":"Create.TimestampedValues/ParDo(ConvertTimestamps)/ParMultiDo(ConvertTimestamps)","hashId":"402#a32dc9f64f1df03a"},{"from":"Create.TimestampedValues/ParDo(ConvertTimestamps)/ParMultiDo(ConvertTimestamps)","to":"Window.Into()/Window.Assign","hashId":"402#8ce970b71df42503"},{"from":"Window.Into()/Window.Assign","to":"Combine.perKey(SumInteger)/GroupByKey","hashId":"402#98f8ba3dc812a76d"},{"from":"Combine.perKey(SumInteger)/GroupByKey","to":"Combine.perKey(SumInteger)/Combine.GroupedValues/ParDo(Anonymous)/ParMultiDo(Anonymous)","hashId":"402#554dcd2b40b5f04b"}]}
```

Comment:
is this line expected to be long single line?

Reply:
Corrected that.